packages/hash/frontend/src/pages/shared/breadcrumbs.tsx

```diff
@@ -54,7 +54,7 @@ const SubMenu = ({ items, defaultIcon }: SubMenuProps) => {
         {items.map((item) => (
           <MenuItem href={item.href} key={item.title}>
             <ListItemIcon>{item.icon ?? defaultIcon}</ListItemIcon>
-            <ListItemText>{item.title}</ListItemText>
+            <ListItemText>{item.title || "Untitled"}</ListItemText>
```

Comment:
The default title of "Untitled" is in 3 places now - you may want to save this to a variable

Reply:
Addressed here https://github.com/hashintel/hash/pull/956/commits/c51458079b6dcb1d7e5c9dbc8eeae53a7c5f8e29